iotdb-client/client-py/iotdb/utils/Field.py

```diff
@@ -134,22 +133,19 @@ def get_object_value(self, data_type):
         """
         :param data_type: TSDataType
         """
-        if self.__data_type is None:
+        if self.__data_type is None or self.value is None or self.value is pd.NA:
```

Comment:
Do you mean `np.NaN`? I don't found the calculation about pandas in this file.
```suggestion
        if self.__data_type is None or self.value is None or self.value is np.NaN:
```

Reply:
Not the `np.NaN`.
pd.NA will appear when the dataframe has some null value. For example, there is a dataset looks like this.
<img width="501" alt="image" src="https://github.com/apache/iotdb/assets/25913899/ebff7807-2ddc-4df0-9e91-c45ccdc6d778">
In Python client, pd.NA may need to process.
<img width="1060" alt="image" src="https://github.com/apache/iotdb/assets/25913899/1563bb84-0605-40cc-a1e5-e5f6288b0353">